metal3-dev/run.sh

```diff
@@ -13,12 +15,23 @@ fi
 
 source $(dirname $(dirname $0))/utils.sh
 
+if [ ! -f "ocp/ostest/.openshift_install_state.json" ]; then
+    echo "openshift_install_state does not exist, is the cluster up?"
+    exit 1
+fi
+
+PROVISIONING_HOST_IP=$(jq  -r '."*installconfig.InstallConfig".config.platform.baremetal.provisioningHostIP' < ocp/ostest/.openshift_install_state.json)
```

Comment:
The path to the state file depends on the cluster name, which isn't always "ostest".